students/darrell_h/lesson_6/mailroom_with_exceptions.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/local/bin/python3
+"""This module does blah blah."""
+
+
+import datetime
+import os  # this is required to write files to ~/Downloads. os.path.expanduser
+
+
+##############
+#    DATA    #
+##############
+
+d = {'jack': [100, 200, 300, 400],
```

Comment:
avoid single character variable names.  In python we always like descriptive names.  Another programmer (or yourself) will have no idea what `d` refers too half way down the code